src/coreclr/jit/flowgraph.cpp

```diff
@@ -790,6 +1123,12 @@ GenTreeCall* Compiler::fgGetStaticsCCtorHelper(CORINFO_CLASS_HANDLE cls, CorInfo
         result = gtNewHelperCallNode(helper, type, opModuleIDArg);
     }
 
+    if (IsStaticHelperEligibleForExpansion(result))
+    {
+        // Keep class handle attached to the helper call since it's difficult to restore it
+        // from arguments/EntryPoint
```

Comment:
```suggestion
        // Keep class handle attached to the helper call since it's difficult to restore it.
```
Mostly just avoiding the mention of `EntryPoint` since it is not technically true.